src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -124,7 +123,7 @@
 }
 
 .schedule-list__patient-details,
-.schedule-list__action-list-cell {
```

Comment:
Thought I was just applying this to the `State, Action` column. But I was mistakenly applying this styling to each table cell.

So I wrapped that `State, Action` cell in a `<div class="schedule-list__action-state-name`></div>` and applied these styles to only that instead.

Still not 100% sure why that only caused issues in Firefox though...